tests/dags/providers/provider_api_scripts/test_science_museum.py

```diff
@@ -30,15 +34,33 @@
 def _get_resource_json(json_name):
     with open(os.path.join(RESOURCES, json_name)) as f:
         resource_json = json.load(f)
-    return resource_json
+        return resource_json
+
+
+@pytest.fixture
+def object_data():
+    yield _get_resource_json("object_data.json")
+
+
+@pytest.fixture
+def single_image_data():
+    object_data = _get_resource_json("object_data.json")
+    object_data["attributes"]["multimedia"] = object_data["attributes"]["multimedia"][
+        :1
+    ]
+    yield object_data
+
+
+default_params = {
+    "has_image": 1,
+    "image_license": "CC",
+    "page[size]": 100,
+}
 
 
 def test_get_query_param_default():
-    actual_param = sm._get_query_param()
-    expected_param = {
-        "has_image": 1,
-        "image_license": "CC",
-        "page[size]": 100,
+    actual_param = sm.get_next_query_params({}, **{"year_range": YEAR_RANGES[0]})
```

Comment:
Would you be okay with adding an explicit year range here, similar to the test below? That prevents folks from having to know what `YEAR_RANGES` is while looking at this test, since the data structure is immediately available.